packages/brick-kit/src/core/LocationContext.ts

```diff
@@ -513,6 +514,7 @@ export class LocationContext {
           ...breadcrumb.items,
         ];
       }
+      mountRoutesResult.appBar.noCurrentApp = breadcrumb.noCurrentApp;
```

Comment:
面包屑可能是多级配置的，例如路由 /a, /a/b, /a/b/c 三个层级都可能配置这个 `noCurrentApp`，这里按正常逻辑，应该是父路由设置了 `noCurrentApp: true` 后，子路由不用再设置。所以这里最好改为：

```suggestion
      if (hasOwnProperty(breadcrumb, "noCurrentApp"))
        mountRoutesResult.appBar.noCurrentApp = breadcrumb.noCurrentApp;
```